keps/sig-node/compute-device-assignment.md

```diff
@@ -19,99 +19,73 @@ creation-date: "2018-07-19"
+* As a _Device Vendor_, I manufacture devices and I have deep domain expertise in how to run and monitor them. Because I maintain my own Device Plugin implementation, as well as Device Monitoring Agent, I can provide consumers of my devices an easy way to consume and monitor my devices without requiring open-source contributions. The Device Monitoring Agent doesn't have any dependencies on the Device Plugin, so I can decouple monitoring from device lifecycle management. My Device Monitoring Agent works by periodically querying the `/devices/<ResourceName>` endpoint to discover which devices are being used, and to get the container/pod metadata associated with the metrics:
 
 ![device monitoring architecture](https://user-images.githubusercontent.com/3262098/43926483-44331496-9bdf-11e8-82a0-14b47583b103.png)
 
-### Kubelet Device Lifecycle
-
-* As a cluster operator, I want the Kubelet to have consistent behavior during kubelet restarts, node reboots, and node upgrades regardless of any disk failure/corruption or otherwise unrecoverable node state.  Since the Kubelet now obtains container-device bindings from pod objects, the Kubelet's Device Manager no longer needs to checkpoint these bindings or implement complex reasoning about state upon startup.
 
-### Out-of-tree Experimentation
-
-#### Network Device Plugins
-
-* As a _Device Vendor_, I manufacture network devices.  Even though the Kubelet exposes different interfaces for network setup (CNI), and device lifecycle management (Device Plugin), I can enable consumption of my network devices without requiring communication between my CNI and Device Plugin implementations.  This allows me to independently develop, test, and release my component for network setup, and my component for managing network devices without any dependencies between them. My Device Plugin exposes the set of network devices on the node, but does _not_ perform network setup when `Allocate` is called. Instead, when `CNI_ADD` is called on my CNI plugin, it queries the kubelet's `/pods` endpoint to discover the network devices it should perform network setup for:
-
-![network device architecture](https://user-images.githubusercontent.com/3262098/43926528-65444380-9bdf-11e8-9ac2-49c8ae46d3a1.png)
+## Changes
 
-#### Guaranteed Device Scheduling
+Add a Kubelet endpoint, `/devices/<ResourceName>`, wich returns information about the kubelet's assignment of devices to containers.  It returns a single `PodDeviceList`, which is shown below:
+```golang
```

Comment:
and the response will be created reading local cache from the device manager?

Reply:
added